priv/repo/migrations/20201201051531_add_hide_widget_outside_working_hours_to_accounts.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ChatApi.Repo.Migrations.AddHideWidgetOutsideWorkingHoursToAccounts do
+  use Ecto.Migration
+
+  def change do
+    alter table(:accounts) do
+      add :hide_widget_outside_working_hours, :boolean
```

Comment:
would you guys want this on Account or WidgetSettings? I went with Account bc it's not a JS-config setting, but can totally change

Reply:
hmm the more i think about it the more i think it makes sense to have this on WidgetSettings... mainly because from an API design perspective i'd expect to see this field in a `GET /api/widget_settings` rather than a `GET /api/accounts` situation... what do you think?

(if we move it to WidgetSettings, i'd rename it to just `hide_outside_working_hours`)

Reply:
makes sense to me! 👍